packages/models/src/lib/plugins.ts

```diff
@@ -0,0 +1,116 @@
+import { z } from 'zod';
+
+// Define Zod schema for the PluginMetadata type
+const pluginMetadataSchema = z.object(
+  {
+    slug: z.string({
+      description: 'Unique ID (human-readable, URL-safe)',
+    }),
```

Comment:
We should use `.regex()` validation for all our `slug` fields. You can use `/^[a-z0-9]+(?:-[a-z0-9]+)*$/` (it's what I'm using on the portal backend).

Reply:
added